yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/util/Validation.java

```diff
@@ -23,6 +24,12 @@ public static void notNullOrEmpty(String value, String name) {
         }
     }
 
+    public static <T> void notNullOrEmpty(Collection<T> value, String name) {
```

Comment:
Can we add tests for this 👍 Doesn't look like there's any coverage at the moment